crates/sui-graphql-rpc/src/types/object.rs

```diff
@@ -166,29 +186,38 @@ impl Object {
             .extend()
     }
 
-    /// Objects can either be immutable, shared, owned by an address,
-    /// or are child objects (part of a dynamic field)
-    async fn kind(&self) -> Option<ObjectKind> {
+    /// The owner type of this Object.
```

Comment:
Why not keep the kind nomenclature? I think we should enumerate the possible types of ownership/ being owned (or write doc comments on each of Shared, Parent, AddressOwner and direct the user there)

Reply:
> Why not keep the kind nomenclature?

Could have gone either way to be honest, but the native types use the "owner" terminology, so I decided to standardise on that.

> Regardless, I think we should enumerate the possible types of ownership/ being owned (or write doc comments on each of Shared, Parent, AddressOwner and direct the user there)

+1 to both suggestions.